.github/workflows/code_quality.yaml

```diff
@@ -1,34 +1,30 @@
 name: Check code quality
-
 on: [push]
-
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v4
-      with:
-        repository: 'awslabs/git-secrets'
-        ref: 'master'
-    - name: Install git-secrets
-      run: sudo make install
-    - uses: actions/checkout@v4
-    - uses: actions/setup-python@v3
-      with:
-        python-version: '3.10'
-    - name: Install git-secrets in the repository
-      run: git secrets --install
-    - name: Install git-secrets aws register in the repository 
-      run: git secrets --register-aws
-    - name: Scan aws secrets
-      run: git secrets --scan
-    - name: Install pipenv
-      run: python -m pip install --upgrade pip && pip install pipenv
-    - name: Install deps
-      run: pipenv sync --system --dev
-    - name: Run linter
-      run: flake8 .
-    - name: Check formatting
-      run: black --check . && isort --check .
-    - name: Check types
-      run: mypy ./
+      - uses: actions/checkout@v4
+        with:
+          repository: "awslabs/git-secrets"
+          ref: "master"
+      - name: Install git-secrets
+        run: sudo make install
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
+      - name: Install git-secrets in the repository
+        run: git secrets --install
+      - name: Install git-secrets aws register in the repository
+        run: git secrets --register-aws
+      - name: Scan aws secrets
+        run: git secrets --scan
+      - name: Install pipenv
+        run: python -m pip install --upgrade pip && pip install pipenv
+      - name: Install deps
+        run: pipenv sync --system --dev
+      - name: Check formatting
+        run: ruff check . && isort --check .
```

Comment:
We still using isort, but ruff is also a replacement for it 
https://docs.astral.sh/ruff/faq/#how-does-ruffs-import-sorting-compare-to-isort

Should we remove isort too?

Reply:
I've mentioned on the notes section but I didn't removed `isort` because `ruff` don't support all the rules we have at the moment.